cmd/lxd-to-incus/main.go

```diff
@@ -94,6 +94,7 @@ func (c *cmdMigrate) Run(app *cobra.Command, args []string) error {
 	if source == nil {
 		return fmt.Errorf("No source server could be found")
 	}
+	fmt.Printf("==> Detected: %s", source.Name())
```

Comment:
Shouldn't there be a `\n` at the end? Otherwise the output would be:
```
==> Detected: snap package => Looking for target server
```

Or is that intentional?

Reply:
Oops, I'll add the missing line break

Reply:
And fixed!